package.json

```diff
@@ -29,7 +29,7 @@
     "bump-version": "npx standard-version"
   },
   "lint-staged": {
-    "*(apps/**/*.{js,ts}|packages/!(*golden-test-generator)/**/*.{js,ts})": [
+    "packages/!(*golden-test-generator)/**/*.{js,ts}": [
       "yarn lint",
```

Comment:
I'm wondering if it should be just `"eslint"` as this will run the command for every package. I tried it and it seems to work, but still taking a long time. cc @mirceahasegan

Reply:
Hmm, yes that does cut it down to a fraction of the previous runtime